api/pkg/cli/utils/utils.go

```diff
@@ -0,0 +1,108 @@
+// Copyright © 2021 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/tektoncd/hub/api/pkg/cli/app"
+	"github.com/tektoncd/hub/api/pkg/cli/hub"
+	so "github.com/tektoncd/hub/api/pkg/cli/select_options"
+)
+
+type Options struct {
+	Cli          app.CLI
+	From         string
+	Kind         string
+	HubClient    hub.Client
+	selectOption so.Options
```

Comment:
The elements in the `struct` of `so.Options` are somewhat similar to the elements present in this struct so I guess we can merge both select_options.go and this one. WDYT?